verl/workers/rollout/vllm_rollout/vllm_rollout.py

```diff
@@ -98,6 +99,13 @@ def __init__(self, actor_module: nn.Module, config: DictConfig, tokenizer, model
             raise ValueError('Enable chunked prefill, max_num_batched_tokens is smaller than max_model_len, \
                              please increase max_num_batched_tokens or disable chunked prefill')
 
+        # copy it to avoid secretly modifying the engine config
+        engine_kwargs = OmegaConf.to_container(deepcopy(config.engine_kwargs))
+        if engine_kwargs.get('swap_space', None) is None:
+            # vLLM has its own default `swap_space` value, which can change in the future versions.
```

Comment:
shall we loop over all keys and pop keys with None values?

Reply:
Sure, even better! Code updated.